src/ui/Facet/FacetValues.ts

```diff
@@ -308,7 +195,15 @@ export class FacetValues {
     this.values = values;
   }
 
-  sortValuesDependingOnStatus(excludeLastIndex?: number) {
+  sort(options: ISortFacetValuesOptions) {
+    const { facetValuesOrder, numberOfValues } = options;
+
+    this.values = facetValuesOrder.reorderValuesIfUsingAlphabeticalSort(this.values);
+    this.sortValuesDependingOnStatus(numberOfValues);
+    this.values = facetValuesOrder.reorderValuesIfUsingCustomSort(this.values);
```

Comment:
The key to the PR lies here, moving the alphabetical sort before the status sort (status sort moves selected values to the top, excluded values to the bottom, of the visible list). Prior to the PR, alphabetical sort occurred after, which meant status reorders were lost.